ros_docker_images/templates/docker_images/create_ros2_image.Dockerfile.em

```diff
@@ -42,8 +42,7 @@ RUN apt-get update && apt-get install -q -y \
 RUN apt-get update && apt-get install -y \
     locales \
     && locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LC_ALL en_US.UTF-8
+ENV LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8
```

Comment:
@tfoote It just reduces the number of layers for the docker image. There is a limit to the number of layers (127 in total) and any small optimizations like this give the end user more flexibility. If you think this is unnecessary let me know and I wont bother with the PR.

Cheers